control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1396,6 +1396,22 @@ func (r *reconciler) ensureIngressControllersRemoved(ctx context.Context) (bool,
 	if len(errs) > 0 {
 		return false, fmt.Errorf("failed to delete ingress controllers: %w", errors.NewAggregate(errs))
 	}
+
+	routerPods := &corev1.PodList{}
+	if err := r.uncachedClient.List(ctx, routerPods, &client.ListOptions{Namespace: "openshift-ingress"}); err != nil {
+		return false, fmt.Errorf("failed to list pods under openshift-ingress namespace: %w", err)
+	}
+
+	for i := range routerPods.Items {
+		rp := &routerPods.Items[i]
+		if !rp.DeletionTimestamp.IsZero() {
+			log.Info("Force deleting", "pod", client.ObjectKeyFromObject(rp).String())
+			if err := r.client.Delete(ctx, rp, &client.DeleteOptions{GracePeriodSeconds: pointer.Int64Ptr(0)}); err != nil {
+				errs = append(errs, fmt.Errorf("failed to force delete %s", client.ObjectKeyFromObject(rp).String()))
```

Comment:
This errs is not used anywhere after the for loop

Reply:
Handled the error.